lms/djangoapps/instructor/management/commands/collect_course_forums_data.py

```diff
@@ -33,7 +33,7 @@ def handle(self, *args, **options):
         except InvalidKeyError:
             raise CommandError("The course ID given was invalid")
 
-        file_name = "{course_id}-course-forums.csv".format(course_id=course_id.replace("/", "-"))
+        file_name = "{course_id}-course-forums.csv".format(course_id=str(course_id).replace("/", "-"))
```

Comment:
Creating a `CourseKey` from a deprecated string and then reconverting it back to a string via `str` (and not `to_deprecated_string` or even `unicode`) doesn't feel right.

If we need the string value, can't we just use the value from `args[0]`?

`str(SlashSeparatedCourseKey(args[0]))` _may_ be equivalent to `args[0]` for now, but I don't think we can always guarantee this to be true.